docs/content/how-tos/rules/FL0078.md

```diff
@@ -0,0 +1,33 @@
+---
+title: FL0078
+category: how-to
+hide_menu: true
+---
+
+# NestedFunctionNames (FL0078)
+
+*Introduced in `0.21.1`*
```

Comment:
@webwarrior-ws I forgot this has to be updated before I can merge, it should say 0.21.3